frontend/src/modules/editor/themeEditor/less/editorPresetEdit.less

```diff
@@ -44,6 +44,10 @@
 
         .nameEdit {
           display: none;
+
+          .preset-error {
+            color: red;
```

Comment:
Do we have a color variable for errors?
i think it might be `@alert-color`
https://github.com/adaptlearning/adapt_authoring/blob/master/frontend/src/core/less/colours.less#L33

Reply:
I went with red because that is what is used on scaffold errors which is the design I am matching here (see discussion on issue)

Reply:
Cool, thanks for clarifying.
We should have the same styles for errors and warning in the Tool. But I think this can wait until a front-end refactoring.